contracts/blade/BridgeStorage.sol

```diff
@@ -82,12 +82,14 @@ contract BridgeStorage is ValidatorSetStorage {
     function _verifyBatch(SignedBridgeMessageBatch calldata batch) private {
         require(batch.rootHash != bytes32(0), "EMPTY_BATCH");
 
-        if (batch.sourceChainId == block.chainid) {
-            require(lastCommittedInternal[batch.destinationChainId] + 1 == batch.startId, "INVALID_LAST_COMMITTED");
-            lastCommittedInternal[batch.destinationChainId] = batch.endId;
-        } else {
-            require(lastCommitted[batch.sourceChainId] + 1 == batch.startId, "INVALID_LAST_COMMITTED");
-            lastCommitted[batch.sourceChainId] = batch.endId;
+        if (!batch.isRollback) {
```

Comment:
Why don't we verify rollback batch as well?

If you need to differentiate them somehow, create two functions:
- `_verifyRollbackBatch` - does only chainIds verification and isRollback field.
- `_verifyRegularBatch` - does what it does now plus isRollbackField.

Then have a function like this:
```solidity
function _verifyBatch(SignedBridgeMessageBatch calldata batch) private {
        require(batch.rootHash != bytes32(0), "EMPTY_BATCH");
        if (batch.isRollback) {
              _verifyRollbackBatch(batch)
        } else {
              _verifyRegularBatch(batch)
        }
```